ctl/monitoring/monitoring.go

```diff
@@ -33,8 +33,9 @@ import (
 )
 
 const (
-	patternAccesslog  = "/accesslog"
-	patternMonitoring = "/monitoring"
+	patternAccesslog       = "/accesslog"
+	patternMonitoring      = "/monitoring"
+	patternWorkloadMetrics = "/workloadMetrics"
```

Comment:
```suggestion
	patternWorkloadMetrics = "/workload_metrics"
```

IIUC, http url is not case sensitive